### PR TITLE
check-gui: Raise instead of keeping above

### PR DESCRIPTION
**Pull request by pazdera**
To avoid interference with minecraft.

Related to: https://github.com/KanoComputing/peldins/issues/1757

cc @Ealdwulf @alex5imon

### CODE REVIEW

File: kano_updater/ui/available_window.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def __init__(self):
         self.set_icon_name('kano-updater')
 
         # Make sure this window is always above
```
Comment (alex5imon) [inline kano_updater/ui/available_window.py]: you might want to change the comment as well

**Comment by Ealdwulf** [comment]
RelaunchWindow() also does keep_above, I assume that one is not relevant to check-gui? Otherwise looks good.

File: kano_updater/ui/available_window.py (hunk 1 above)
Comment (pazdera) [inline kano_updater/ui/available_window.py]: Good point Alej, thanks!

**Comment by pazdera** [comment]
@Ealdwulf Relaunch window is a sort-of interim window that is shown when the updater relaunches itself during the update phase. There it's actually useful, because we want to make sure to cover everything else with that.

**Comment by tombettany** [comment]
Looks fine to me

**Pull request merged**